src/client/containers/DeleteBoardModal/DeleteBoardModal.container.js

```diff
@@ -5,17 +5,25 @@ import CloseButton from '../../components/CloseButton/CloseButton.component';
 import PageTitle from '../../components/PageTitle/PageTitle.component';
 import GenericButton from '../../components/GenericButton/GenericButton.component';
 
-const DeleteBoardModal = ({ boardInfo, setModalVisibility }) => {
-  const userId = 2;
-  const boardId = 1;
+const DeleteBoardModal = ({
+  boardInfo,
+  setModalVisibility,
+  userId,
+  deleteErrorHandler,
+}) => {
+  const boardId = boardInfo.id;
+
   const deleteBoard = async () => {
     const API_URL = `/api/users/${userId}/boards/${boardId}`;
     try {
-      await fetch(API_URL, { method: 'DELETE' });
+      const response = await fetch(API_URL, { method: 'DELETE' });
       // eslint-disable-next-line no-alert
+      if (!response.ok) {
+        throw new Error(response.status);
+      }
```

Comment:
I think we're good if the alert is removed. You will have to find an other way to notify the user